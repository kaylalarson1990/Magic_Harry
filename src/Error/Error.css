.page-not-found {
    font-size: 48px;
    color: #BE0125;
}