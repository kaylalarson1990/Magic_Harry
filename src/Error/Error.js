import React from "react";
import "./Error.css";
import sortinghat from "../images/sortinghat.png";

const Error = () => {
  return (
    <section>
      <h2 className="page-not-found">
        404 ERROR: PAGE NOT FOUND{" "}
        <img src={sortinghat} alt="wand" className="sorting-hat" />
      </h2>
    </section>
  );
};

export default Error;
