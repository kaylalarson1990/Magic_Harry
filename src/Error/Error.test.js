import React from "react";
import { shallow } from "enzyme";
import Error from "./Error";

describe("Error", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Error />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
