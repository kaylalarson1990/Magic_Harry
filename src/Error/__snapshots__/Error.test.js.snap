// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match snapshot 1`] = `
<section>
  <h2
    className="page-not-found"
  >
    404 ERROR: PAGE NOT FOUND
     
    <img
      alt="wand"
      className="sorting-hat"
      src="sortinghat.png"
    />
  </h2>
</section>
`;
